src/types/catalog.d.ts

```diff
@@ -14,6 +14,236 @@ import {
 
 export default Catalog;
 
+export interface CreateOrReplaceItemsParameters {
+  items: Item[];
+  force?: boolean;
+  notificationEmail?: string;
+  section?: string;
+}
+
+export interface UpdateItemParameters extends CreateOrReplaceItemsParameters {}
+
+export interface DeleteItemsParameters {
+  items: Pick<Item, 'id'>[];
+  section?: string;
+  notificationEmail?: string;
+}
+
+export interface RetrieveItemsParameters {
+  ids?: string[];
+  section?: string;
+  numResultsPerPage?: number;
+  page?: number;
+}
+
+export interface CreateOrReplaceVariationsParameters {
+  variations: Variation[];
+  force?: boolean;
+  notificationEmail?: string;
+  section?: string;
+}
+
+export interface UpdateVariationsParameters
+  extends CreateOrReplaceVariationsParameters {}
+
+export interface DeleteVariationsParameters {
+  variations: Pick<Variation, 'id'>[];
+  force?: boolean;
+  notificationEmail?: string;
+  section?: string;
+}
+
+export interface RetrieveVariationsParameters {
+  section?: string;
+  ids?: string[];
+  itemId?: string;
+  numResultsPerPage?: number;
+  page?: number;
+}
+
+export interface AddItemGroupParameters {
+  id: string;
+  name: string;
+  parent_id?: string;
```

Comment:
I think for node, with the pr from 1 month ago, we changed all the parameters to camel case. Should we change the types as well?